transport-classes-epoll/src/main/java/io/netty5/channel/epoll/EpollSocketChannel.java

```diff
@@ -501,6 +501,11 @@ private final class EpollSocketWritableByteChannel extends SocketWritableByteCha
             super(socket);
         }
 
+        @Override
+        protected int write(final ByteBuffer buf, final int pos, final int limit) throws IOException {
+            return socket.send(buf, pos, limit);
```

Comment:
I believe we can make better here for Netty 5 ie why not using generics for the type of the base class here;
this way, fd will be the right time without using tricks like these

Reply:
@franz1981 want to add a commit on top of this in the `recv_send_port` branch ?